ChangeLog

```diff
@@ -1,3 +1,5 @@
+	* implemented support magnet URI extension, select specific file indices
+	  for download, BEP53
```

Comment:
would you mind adding a bullet for this BEP on the ``docs/features.rst`` page as well? ideally with a link to bittorrent.org

Reply:
OK